crates/astria-sequencer/src/app/mod.rs

```diff
@@ -1024,10 +1025,11 @@ impl App {
 
         // flag mempool for cleaning if we ran a fee change action
         self.recost_mempool = self.recost_mempool
-            || signed_tx
-                .actions()
-                .iter()
-                .any(|action| matches!(action, Action::FeeAssetChange(_) | Action::FeeChange(_)));
+            || matches!(signed_tx.group(), Some(ActionGroup::BundlableSudo(_)))
```

Comment:
It's convetion to create inherent methods for checks like this (similar to the check below; I think I commented this before?):

```rust
impl SignedTransaction {
    // NOTE: I used `contains_*` rather than `is_` because it read weird; maybe you can come up with better naming?
    fn contains_bundleable_sudo_actions(&self) -> bool {
        matches!(self.group(), ActionGroup::BundleSudo(_))
    }
}
```